docs/main/user-guides/upgrades.md

```diff
@@ -126,8 +131,19 @@ openebs-upgrade-v2-6-0-s58xl                   0/1     Completed   0          7m
 
 4. Once the upgrade process completes, all the volumes and pools should be online.
 
+5. Users who had opted for hot upgrade and disabled the partial rebuild using `--set mayastor.agents.core.rebuild.partial.enabled=true`, can enable it back with the below command.
```

Comment:
Modify the sentence as per the above comment

Reply:
We have modified the sentence like this: 

If you have disabled the partial rebuild during upgrade, re-enable it by adding the value `--set mayastor.agents.core.rebuild.partial.enabled=true` in the upgrade command.

Hope this looks fine @avishnu